pkg/cmd/marathon/marathon_ping.go

```diff
@@ -22,3 +38,27 @@ func newCmdMarathonPing(ctx api.Context) *cobra.Command {
 
 	return cmd
 }
+
+func marathonPing(ctx api.Context, client marathon.Client, once bool) (int, error) {
+	numberOfMasters := 1
+	if !once {
+		cluster, err := ctx.Cluster()
+		if err != nil {
+			return 0, err
+		}
+		mesosClient := mesos.NewClient(pluginutil.HTTPClient(cluster.URL()))
+		masters, err := mesosClient.Masters()
+		if err != nil {
+			return 0, err
+		}
+		numberOfMasters = len(masters)
+	}
+
+	for i := 1; i <= numberOfMasters; i++ {
+		_, err := client.API.Ping()
+		if err != nil {
+			return 0, fmt.Errorf("Unable to ping leading Marathon master %d time(s)", i)
```

Comment:
I've been seeing errors returned without capitalization in our other projects. @makkes did we decide that error messages should be uncapitalized?

Reply:
I can see that we mostly do not capitalize the first letter in dcos-core-cli but we have some inconsistencies, removing the capital now.